### DISCUSSION TITLE
new version(1.7.0) fails to load ~/.ideavimrc

### DISCUSSION THREAD

**Discussion by astroshot** [JetBrains/ideavim#365] [General]
I update ideavim to the newest version 1.7.0, but I discover that idea fails to load ~/.ideavimrc. It doesn't work after sevaral restart of my idea. I have to rollback to 0.69 version, and it works well. 

So any help?
 
Here is my idea version:
```text
IntelliJ IDEA 2020.3.4 (Community Edition)
Build #IC-203.8084.24, built on April 27, 2021
Runtime version: 11.0.10+8-b1145.96 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.16
GC: ParNew, ConcurrentMarkSweep
Memory: 2014M
Cores: 4
Non-Bundled Plugins: IdeaVIM, com.tylerthrailkill.intellij.solarized, com.werfad, net.vektah.codeglance, izhangzhihao.rainbow.brackets, zielu.gittoolbox, String Manipulation, com.dubreuia, MavenRunHelper
```

and ideavimrc:

```vim
"
"          _________ _______   _________ _         _________ ______   _______  _______
" |\     /|\__   __/(       )  \__   __/( (    /|  \__   __/(  __  \ (  ____ \(  ___  )
" | )   ( |   ) (   | () () |     ) (   |  \  ( |     ) (   | (  \  )| (    \/| (   ) |
" | |   | |   | |   | || || |     | |   |   \ | |     | |   | |   ) || (__    | (___) |
" ( (   ) )   | |   | |(_)| |     | |   | (\ \) |     | |   | |   | ||  __)   |  ___  |
"  \ \_/ /    | |   | |   | |     | |   | | \   |     | |   | |   ) || (      | (   ) |
"   \   /  ___) (___| )   ( |  ___) (___| )  \  |  ___) (___| (__/  )| (____/\| )   ( |
"    \_/   \_______/|/     \|  \_______/|/    )_)  \_______/(______/ (_______/|/     \|
"

" =====================================================================================
" In IDEA, type `:actionlist` to query all action commands
" =====================================================================================
" Common Settings
set nu
set rnu
let mapleader = ","
nnoremap ; :
set clipboard+=unnamed

" Emulated Vim Plugins
set surround
set showmode
set multiple-cursors
set incsearch
set easymotion

" Easy Motion Based on KJump
nmap <leader><leader>s :action KJumpAction<cr>
nmap <leader><leader>w :action KJumpAction.Word0<cr>
nmap <leader><leader>j :action KJumpAction.Line<cr>

" IDEA Settings
set ideajoin

" Open Annotate
nmap <leader>gb :action Annotate<CR>
nmap <leader>rn :action RenameElement<CR>
nmap <leader>rf :action ReformatCode<CR>
nmap <leader>ri :action OptimizeImports<CR>

```

**Comment by astroshot** [comment]
I droped those comment lines at the first of ideavimrc, and it works 😂

**Comment by mogita** [comment]
I've seen similar behavior in 1.7.1, my `~/.ideavimrc` looks like this:

```
source ~/.vimrc
```

All my actual Vim configs are inside the `~/.vimrc` file and I use them everyday in the Terminal and GoLand before the recent plugin update, from 0.6.x to 1.7.1 I believe. But after the update things seemed to be broken.

I tried removing any leading comments as @astroshot suggested but no luck.

**Reply by AlexPl292** [reply]
Could you please share your .vimrc?

**Reply by mogita** [reply]
Here's my `.vimrc`. I've customized it a little from the original work.

https://gist.github.com/mogita/f39a5ce032412cc63601436933b05cf4

**Reply by AlexPl292** [reply]
@lippfi could you please check it?

**Reply by lippfi** [reply]
> Here's my `.vimrc`. I've customized it a little from the original work.
> 
> https://gist.github.com/mogita/f39a5ce032412cc63601436933b05cf4

Our grammar requires space after `execute` keyword (it will be fixed in future release). For now you should replace line 399 `        execute("bdelete! ".l:currentBufNum)` with `execute ("bdelete! ".l:currentBufNum)`

**Reply by mogita** [reply]
This fixes the issue. A big thank you!
